matching-core/src/main/java/com/graphhopper/matching/util/HmmProbabilities.java

```diff
@@ -67,33 +55,12 @@ public double emissionLogProbability(double distance) {
      *                       consecutive map matching candidates.
      * @param linearDistance Linear distance [m] between two consecutive GPS
      *                       measurements.
-     * @param timeDiff       time difference [s] between two consecutive GPS
-     *                       measurements.
      */
-    public double transitionLogProbability(double routeLength, double linearDistance,
-                                           double timeDiff) {
-        if (timeDiff == 0) {
```

Comment:
As discussed [here](https://github.com/graphhopper/map-matching/pull/83#issuecomment-265827142) I'd prefer a speed based limit where we break sequences (#87) if the speed between two candidates is too high. However, I think we can leave that until after this PR is merged.